scripts/travis/travis-tests

```diff
@@ -161,6 +161,9 @@ ip net add test
 ./test/zdtm.py run -t zdtm/static/env00 -k always
 ./test/crit-recode.py
 
+# libcriu testing
+./test/others/libcriu/run.sh
+
```

Comment:
There are a few tests listed from the `other` folder above. You can add something like:
```
make -C test/others/libcriu/ run
```

Reply:
Right, that should prevent the errors I am seeing. Thanks for the suggestion.